src/Autofac.Multitenant/MultitenantContainer.cs

```diff
@@ -53,13 +53,24 @@ namespace Autofac.Multitenant
     /// that is passed into the container during construction.
     /// </para>
     /// <para>
-    /// Tenant lifetime scopes are immutable, so once they are retrieved,
+    /// The ability to remove (<see cref="RemoveTenant(object)"/>) or reconfigure
+    /// (<see cref="ReconfigureTenant(object, Action{ContainerBuilder})"/> an
+    /// active tenant has been added.  However, it must still be noted that
```

Comment:
"...an active tenant exists." - long term it won't have been added, it just exists.